histomicstk/cli/NucleiDetection/NucleiDetection.xml

```diff
@@ -192,6 +192,38 @@
       <longflag>min_fgnd_frac</longflag>
       <default>0.25</default>
     </double>
+  </parameters>
+    <parameters advanced="true">
+    <label>Frame and Style</label>
+    <description>Frame parameters</description>
+    <string>
+      <name>frame</name>
+      <longflag>frame</longflag>
+      <label>Frame Index</label>
+      <description>Frame index in a multi-frame image</description>
+      <default>{#control:#current_image_frame#}</default>
+    </string>
+    <string>
+      <name>style</name>
+      <longflag>style</longflag>
+      <label>Style Options</label>
+      <description>Image style options for compositing a multi-frame image</description>
+      <default>{#control:#current_image_style#}</default>
+    </string>
+  </parameters>
+  <parameters advanced="true">
+    <label>Image inversion</label>
+    <description>Choose if image inversion is needed</description>
+    <string-enumeration>
+      <name>ImageInversionForm</name>
+      <description>Image inversion may be needed for greyscale images with bright nuclei and dark background. default option will automatically invert the image if it is sinlge channel. Choose if color inversion is needed.</description>
+      <channel>input</channel>
+      <longflag>output_form</longflag>
```

Comment:
We probably want to rename this, as it isn't related to the output.  Maybe `invert_image` instead of `output_form`.